### PR TITLE
add new extension screnshot full page in browser

### PR DESCRIPTION
**Pull request by Coun1er**
## Description

This extension allows you to quickly make a screenshot of the entire page of the site and save it to your desktop just by writing the address of the site.

## Checklist

- [*] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [*] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [*] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [*] I checked that files in the `assets` folder are used by the extension itself
- [*] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by Coun1er** [comment]
I do not understand what this error means?

File: extensions/screenshot-full-page-in-browser/src/screenshot-full-page.tsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+      toast.style = Toast.Style.Success;
+      toast.title = "Took screenshot of website";
+      toast.message = "Look for it on the desktop"
+      toast.primaryAction = {
+        title: "Show in Finder",
+        onAction: async () => {
+          await showInFinder(outputFile);
+        },
+      };
+      await setTimeout(2000);
```
Comment (pernielsentikaer) [inline extensions/screenshot-full-page-in-browser/src/screenshot-full-page.tsx:71]:
```suggestion
```
I think it's better to clear the input field and leave the extension open.

File: extensions/screenshot-full-page-in-browser/src/screenshot-full-page.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+      });
+      return;
+    }
+
+    const toast = await showToast({ style: Toast.Style.Animated, title: "Taking screenshot of website..." });
+
+    try {
+      let webSite;
+      if (!values.website.startsWith("http")) {
+        webSite = "http://" + values.website;
```
Comment (pernielsentikaer) [inline extensions/screenshot-full-page-in-browser/src/screenshot-full-page.tsx:42]:
```suggestion
        webSite = "https://" + values.website;
```

File: extensions/screenshot-full-page-in-browser/package.json [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  "categories": [
+    "Web",
+    "Developer Tools",
+    "Productivity"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "screenshot-full-page",
+      "title": "Screenshot full page in browser",
```
Comment (pernielsentikaer) [inline extensions/screenshot-full-page-in-browser/package.json:17]:
```suggestion
      "title": "Capture Fullpage Screenshot of Website",
```
Could this be a better title?

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @Coun1er 👋 

Thanks for your contribution 💪 

I have some suggestions and comments for you address 🙂

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

File: extensions/screenshot-full-page-in-browser/src/screenshot-full-page.tsx (hunk 1 above)
Comment (Coun1er) [reply extensions/screenshot-full-page-in-browser/src/screenshot-full-page.tsx:71]: `const webSite = useState<string>();`
`(webSite) => {
        current?.reset();
      }`
      or
`(webSite) => {
        reset();
      }`
doesn't work.
Sorry. I want to reset this field, but I can't figure out how to use this method:
[https://yadi.sk/i/QQcaXDjjXr_nNQ](url)

Comment (pernielsentikaer) [reply extensions/screenshot-full-page-in-browser/src/screenshot-full-page.tsx:71]: I'll do the change for this one and you can have a look 🔥

**Review by Coun1er — COMMENTED** [review]
Damn) I did, but it didn't work through useRef)

**Review by Coun1er — COMMENTED** [review]
I am very ashamed that you are correcting so much for me. Thank you!

**Comment by Coun1er** [comment]
Hi, is there anything else I need to do to be added to the store? A few of my colleagues are waiting to use it.

**Review by pernielsentikaer — APPROVED** [review]
Lets get it merged 🔥

**Pull request merged**